src/hooks/todayTile/useTodayTasks.tsx

```diff
@@ -0,0 +1,126 @@
+import { useCallback } from 'react';
+import { useGraphQLClient } from '../useGraphQLClient';
+import { gql } from 'graphql-request';
+import { useActiveAccount } from '../useActiveAccount';
+import { useQuery } from '@tanstack/react-query';
+import { useActiveProject } from '../useActiveProject';
+import { useRestQuery } from '../rest-api';
+import { ConsentTask, SurveyResponse } from './types';
+
+export type TodayTask = SurveyResponse | ConsentTask;
+export function isConsentTask(value: TodayTask): value is ConsentTask {
+  return !!(value as ConsentTask).form;
+}
+
+const useConsentTasks = () => {
+  const { activeProject } = useActiveProject();
+  const { accountHeaders } = useActiveAccount();
+
+  return useRestQuery(
+    'GET /v1/consent/directives/me',
+    {
+      projectId: activeProject?.id!,
+      includeForm: true,
+    },
+    {
+      enabled: !!accountHeaders && !!activeProject,
+      axios: { headers: accountHeaders },
+      select: (data) => data.items,
+    },
+  );
+};
+
+const useGetSurveyResponsesForProject = () => {
+  const { activeSubjectId, activeProject } = useActiveProject();
+  const { accountHeaders } = useActiveAccount();
+
+  return useRestQuery(
+    'GET /v1/survey/projects/:projectId/responses',
+    {
+      projectId: activeProject!.id,
+      author: activeSubjectId!,
+      patientId: activeSubjectId!,
+      includeSurveyName: true,
```

Comment:
Do we need the survey name?

Reply:
No, we don't actually need this to display the badge, I've updated this to false.